apps/system/style/app_titlebar.css

```diff
@@ -61,6 +61,10 @@
   position: absolute;
 }
 
+.appWindow:not(.collapsible) .chrome + .titlebar .titlebar-minimized {
```

Comment:
Since we're touching this, can you add a comment about what the 11.5 rem is for?